tests/pr2_jacobian_tests/src/main.cpp

```diff
@@ -1,193 +0,0 @@
-/*********************************************************************
```

Comment:
Why not keep these tests and just make them work with Catkin?

Reply:
The email response didn't get the message to the right place, see https://github.com/ros-planning/moveit_pr2/pull/96#issuecomment-382352007 for reply.